src/functions/cronTrigger.js

```diff
@@ -86,6 +87,12 @@ export async function processCronTrigger(event) {
         res: {},
       }
     }
+    // Send Telegram message on monitor change
+    if (monitorsState[monitor.id].operational !== monitorOperational
```

Comment:
```suggestion
    if (monitorStatusChanged
```

Reply:
this should resolve the issue after changes in develop :)

Reply:
Ohh I see, thanks!